src/google_cloud_debugger/google_cloud_debugger_lib/dbg_breakpoint.cc

```diff
@@ -142,9 +146,49 @@ bool DbgBreakpoint::TrySetBreakpoint(
   return best_match_index != -1;
 }
 
+HRESULT DbgBreakpoint::EvaluateExpressions(DbgStackFrame *stack_frame,
```

Comment:
I think we should ignore all collection restrictions when evaluation expressions.   Such as DbgObject::collection_size_ and buffer sizes (up to the hard limit).

Reply:
I'll address this in the next PR.